src/Goerli-DssSpellCollateral.sol

```diff
@@ -56,41 +49,41 @@ contract DssSpellCollateralAction {
         //  Poll Link 2: https://vote.makerdao.com/polling/QmS7dBuQ
         //  Forum Post:  https://forum.makerdao.com/t/reth-collateral-onboarding-risk-evaluation/15286
 
-        DssExecLib.addNewCollateral(
-            CollateralOpts({
-                ilk:                  "RETH-A",
-                gem:                  RETH,
-                join:                 MCD_JOIN_RETH_A,
-                clip:                 MCD_CLIP_RETH_A,
-                calc:                 MCD_CLIP_CALC_RETH_A,
-                pip:                  PIP_RETH,
-                isLiquidatable:       true,
-                isOSM:                true,
-                whitelistOSM:         true,
-                ilkDebtCeiling:       0,                 // line updated to 0 (previously 5M)
-                minVaultAmount:       15_000,            // debt floor - dust in DAI
-                maxLiquidationAmount: 2_000_000,
-                liquidationPenalty:   13_00,             // 13% penalty on liquidation
-                ilkStabilityFee:      ONE_FIVE_PCT_RATE, // 1.50% stability fee
-                startingPriceFactor:  110_00,            // Auction price begins at 110% of oracle price
-                breakerTolerance:     50_00,             // Allows for a 50% hourly price drop before disabling liquidation
-                auctionDuration:      120 minutes,
-                permittedDrop:        45_00,             // 45% price drop before reset
-                liquidationRatio:     170_00,            // 170% collateralization
-                kprFlatReward:        250,               // 250 DAI tip - flat fee per kpr
-                kprPctReward:         10                 // 0.1% chip - per kpr
-            })
-        );
-
-        DssExecLib.setStairstepExponentialDecrease(MCD_CLIP_CALC_RETH_A, 90 seconds, 99_00);
+        // DssExecLib.addNewCollateral(
```

Comment:
There are more generalized default comments we should swap out here: https://github.com/makerdao/spells-mainnet/blob/master/src/DssSpellCollateral.sol

Reply:
Done in [f52b7eb](https://github.com/makerdao/spells-goerli/pull/131/commits/f52b7eb8005d65f0b59e6a9fd2ebbe51a8592aac)